### PR TITLE
Add more keywords:"flag","smile",etc & fix mistakes

### PR DESCRIPTION
**Pull request by iViolinSolo**
The website is perfect, so brilliant, I love it so much. So, I add more tags for it.

### CODE REVIEW

**Comment by muan** [comment]
I think cartoon is a bit too generic to be a keyword. :/ everything emoji could be considered a cartoon really.

**Comment by iViolinSolo** [comment]
@muan aha~I think you are right. I will change it latter.

**Comment by iViolinSolo** [comment]
@muan  Now I have remove the keyword "cartoon", and add some new keywords. Waiting for your suggesstions~ :blush:

File: emojis.json [hunk 1]
```diff
@@ -188,14 +188,14 @@
...
-  "gem": ["blue", "ruby"],
+  "gem": ["blue", "ruby", "diamond"],
   "bust_in_silhouette": ["man", "user", "person", "human"],
   "busts_in_silhouette": ["man", "user", "person", "human", "group", "team"],
   "speech_balloon": ["bubble", "words"],
   "footprints": ["feet", "tracking"],
   "thought_balloon": ["bubble", "cloud", "speech"],
   "dog": ["animal", "friend", "nature", "woof"],
-  "wolf": ["animal", "nature"],
+  "wolf": ["animal", "nature",
```
Comment (yous) [inline emojis.json]: `]` is removed.

**Comment by yous** [comment]
Rebase is needed due to merge conflicts.

**Comment by iViolinSolo** [comment]
Ahm~ Seems I'm careless. And here're the latest version.

**Comment by yous** [comment]
:+1:

**Comment by iViolinSolo** [comment]
@yous  :smile:  THX

**Comment by muan** [comment]
Thanks a ton both @yous and @BigDipper7 :zap: Mergin! :zap:

**Pull request merged**